modules/kubernetes-addons/prometheus/locals.tf

```diff
@@ -72,10 +47,36 @@ locals {
     operating_system = "linux",
   })]
 
+  amazon_prometheus_workspace_id           = var.amazon_prometheus_workspace_id != null && var.enable_amp_for_prometheus ? var.amazon_prometheus_workspace_id : null
+  amazon_prometheus_workspace_url          = var.enable_amp_for_prometheus && var.amazon_prometheus_workspace_id != null ? "https://aps-workspaces.${data.aws_region.current.id}.amazonaws.com/workspaces/${local.amazon_prometheus_workspace_id}/api/v1/remote_write" : null
```

Comment:
Is this value the same as `amazon_prometheus_workspace_endpoint`? If so, should we be passing that value instead of the workspace_id?

Reply:
Not sure. I will try to print out to see what comes out